tests/net/socket/select/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+common:
+  depends_on: netif
+  platform_whitelist: native_posix qemu_x86 qemu_cortex_m3
+tests:
+  net.socket:
+    min_ram: 21
+    tags: net
```

Comment:
I suggest to add "socket" tag to all socket related tests, then it is easier to just run socket tests if needed.

Reply:
> I suggest to add "socket" tag to all socket related tests, then it is easier to just run socket tests if needed.

Sounds good, but probably as a separate patch? Would be a good fit for #12648.

Reply:
Yes, separate PR like #12648 would be fine.

Reply:
> I suggest to add "socket" tag

Ok, pushed addition of the tag to exiting tests in #12648, then included here in new tests too.